src/test/java/uk/co/ramp/policy/isolation/SingleCaseIsolationPolicyTest.java

```diff
@@ -125,10 +124,7 @@
                       .build())
               .build();
 
-  private final BoundedDistribution flatThreeDays = mock(BoundedDistribution.class);
-
-  private final BoundedDistribution thresholdLinearBoundedDistribution =
-      mock(BoundedDistribution.class);
+  private BoundedDistribution thresholdLinearBoundedDistribution = mock(BoundedDistribution.class);
```

Comment:
The initialization of `thresholdLinearBoundedDistribution` doesn't make sense to me.  The initial `mock(BoundedDistribution.class)` set here is never used.  It is set to another value in `setUp()`.  (See also my comments below...)